### ISSUE TITLE
6032 100nF tantalum

### ISSUE THREAD

**Issue by riccardo92** [ehjortberg/kicad-p1-port-thingie#1]
I cannot seem to find any 100nF SMD tantalum capacitors in 6032 package. For example, even on kemet.com, applying filters for 0.1uF and 6032 will not give any results. Same for Amazon, Aliexpress etc.

Where can I buy these capacitors? Is there an alternative I can use? Thanks in advance. I'm in Europe btw.

Already had the PCBs made, so redesigning the PCB slightly would be a bit of a shame.

**Comment by ehjortberg** [comment]
Hi! Happy that you have chosen to use my build!

I have used Vishay 293D104X9035A2TE3 from tme.eu last time I built one.
They are apparently 1206/3216 but fit well.
The datasheet for the LD1117 actually says 100nF on the Vin and 10uF on Vout but that is not that crucial in this application. So basically anything in the ballpark will work.
I will try to update the instruction when there is some time.
